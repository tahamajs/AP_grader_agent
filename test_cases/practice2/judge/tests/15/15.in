enter 1
reserve_table 1
enter 2
reserve_table 2
enter 3
reserve_table 3
enter 22
reserve_table 22
enter 5
reserve_table 5
enter 4
reserve_table 4
enter 24
reserve_table 24
enter 35
reserve_table 35
enter 6
reserve_table 6
enter 46
reserve_table 46
enter 49
reserve_table 49
