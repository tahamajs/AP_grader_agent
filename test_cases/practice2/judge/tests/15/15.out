Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Paria sits at table 1
Table 1: 3 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Amir Hossein sits at table 1
Table 1: 2 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Parsa sits at table 1
Table 1: 1 0
Table 2: 4 0
Table 5: 3 0
Table 3: 2 0
Table 6: 3 0
Table 4: 2 0
Kiana sits at table 1
Table 1: 0 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Fereshte enters the waiting queue of table 1
Table 1: 0 1
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Mehrad enters the waiting queue of table 1
Table 1: 0 2
Table 2: 4 0
Table 5: 3 0
Table 3: 2 0
Table 6: 3 0
Table 4: 2 0
Sara enters the waiting queue of table 1
Table 1: 0 3
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Adolf enters the waiting queue of table 1
Table 2: 4 0
Table 6: 3 0
Table 1: 0 4
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Amir Akbar sits at table 2
Table 2: 3 0
Table 6: 3 0
Table 1: 0 4
Table 5: 3 0
Table 4: 2 0
Table 3: 2 0
Shahin sits at table 2
Table 2: 2 0
Table 1: 0 4
Table 6: 3 0
Table 4: 2 0
Table 5: 3 0
Table 3: 2 0
Parham sits at table 2
